### PR TITLE
Update CLI docs

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Update CLI docs

Preview URL: https://update-cli-docs.botkube-docs.pages.dev/next/cli/getting-started

## Related issue(s)

- https://github.com/kubeshop/botkube/issues/1125

### CODE REVIEW

File: magefiles/target/sync-cli-docs.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+	"botkube.io/tools/printer"
+	"botkube.io/tools/shellx"
+)
+
+const (
+	cliDocsDstPath = "docs/cli/commands"
+
+	// TODO: revert before merge
+	//cliDocsSrcURL  = "github.com/kubeshop/botkube//cmd/cli/docs?ref=main"
```
Comment (mszostok) [inline magefiles/target/sync-cli-docs.go:21]: after merging https://github.com/kubeshop/botkube/pull/1131/files

File: docs/cli/migrate.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+---
+id: migrate
+title: "Migrating from Botkube Open Source"
+sidebar_position: 2
+---
+
+## Migrating from Botkube Open Source
```
Comment (mszostok) [inline docs/cli/migrate.md:7]: Maybe it should be under some "guides" or even "cloud" category? or I should change the title?

Comment (pkosiec) [reply docs/cli/migrate.md:7]: 🤔 let's postpone this decision for now, I think it is OK as it is right now unless we have more docs related to Cloud 👍

Comment (pkosiec) [inline docs/cli/migrate.md:7]: WDYT about "migrating to Cloud" instead from "OS"? At the end Botkube connected to Cloud is still the open source one. Something like:

```suggestion
id: migrating-installation-to-botkube-cloud
title: "Migrating installation to Botkube Cloud"
sidebar_position: 2
---

## Migrating installation to Botkube Cloud
```

File: docs/cli/getting-started.mdx
Comment (pkosiec) [inline docs/cli/getting-started.mdx:1]: Please make sure the version specified here will be replaced by release job.
Currently we do:

```bash
      - name: Update versions in installation docs
        run: |
          LATEST_RELEASE_VERSION="${{ inputs.version }}.0"
          PREVIOUS_RELEASE_VERSION_WITH_PREFIX=$(grep  "\--version" docs/installation/slack/index.md | head -1 | awk -F'--version ' '{print $2}' | awk -F' ' '{print $1}')
          PREVIOUS_RELEASE_VERSION=${PREVIOUS_RELEASE_VERSION_WITH_PREFIX:1}
          find ./docs/installation/ -type f -name "*.md" -exec sed -i.bak "s/$PREVIOUS_RELEASE_VERSION/$LATEST_RELEASE_VERSION/g" {} \;
          find ./docs/configuration/ -type f -name "*.md" -exec sed -i.bak "s/$PREVIOUS_RELEASE_VERSION/$LATEST_RELEASE_VERSION/g" {} \;
```

**Review by pkosiec — APPROVED** [review]
LGTM with minor comments 👍

**Pull request merged**